src/User.php

```diff
@@ -5511,7 +5511,7 @@ public function getAuthToken($field = 'personal_token', $force_new = false)
 
        // check date validity for cookie token
         $outdated = false;
-        if ($field === 'cookie_token') {
+        if ($field === 'cookie_token' && !empty($this->fields[$field . "_date"])) {
```

Comment:
Maybe the token should be considered as outdated if it has no expirationd ate.

Reply:
I really don't know. @orthagh?